classes/ProjectData.php

```diff
@@ -365,5 +365,20 @@ public static function getCriticalQuestions1LogicForMissing($question_1){
         }
         return $logic;
     }
+
+    public static function getChoiceLabelsArray($module, $study, $project_id){
+        $type = getFieldType($study, $project_id);
+        $choicesById = [];
+        if($type == 'truefalse') {
+            $choicesById[0] = "False";
+            $choicesById[1] = "True";
+        } else if ($type == 'yesno') {
+            $choicesById[0] = "No";
+            $choicesById[1] = "Yes";
```

Comment:
@knil-maloon are these conditions not included in `gteChoiceLabels`?

Reply:
Heh. I just replied to this on my comment before.
No, it seems these two options are not added there.